components/ws-proxy/pkg/sshproxy/server.go

```diff
@@ -545,9 +545,11 @@ func (s *Server) GenerateSSHCert(ctx context.Context, userName string) (ssh.Sign
 		CertType:        ssh.UserCert,
 		Permissions: ssh.Permissions{
 			Extensions: map[string]string{
-				"permit-pty":            "",
-				"permit-user-rc":        "",
-				"permit-X11-forwarding": "",
+				"permit-pty":              "",
```

Comment:
Is here some security concern? Or it is artificial limitations? We did a breaking change but limiting it.

Reply:
I understand, but is it really good for us? if people have some unexpecting ssh config they won't be able to connect at all with any integration?

Reply:
I mean if someone is using force command it will be rejected now too? Or you think we cover all cases?